src/preprocess/fastMRI_preprocessing.py

```diff
@@ -0,0 +1,84 @@
+import h5py
+import logging
+import argparse
+import xmltodict
+import numpy as np
+import nibabel as nib
+from tqdm import tqdm
+from pathlib import Path
+import SimpleITK as sitk
+
+
+def main(args):
+    input_dir = args.input_dir
+    output_dir = args.output_dir
+    
+    if output_dir.exists() is False:
+        output_dir.mkdir(parents=True)
+        
+    data_paths = sorted(list(input_dir.iterdir()))
+    min_size = np.array([1e10, 1e10, 1e10])
+    for path in tqdm(data_paths):
+        filename = path.parts[-1].split('.')[0]
+        mri_type = filename.split('_')[2]
+        sid = filename.split('_')[-1]
+
+        output_subdir = output_dir / mri_type
+        if output_subdir.exists() == False:
```

Comment:
Maybe `if output_subdir.exists() is False:` or `if not output_subdir.exists():`.

Reply:
Modified to `if output_subdir.exists() is False`.
But I don't understand the difference between them.